docs/4.0/examples/tests/input-group.md

```diff
@@ -0,0 +1,462 @@
+---
+layout: default
+title: Test - Input groups
+sizes:
+  - "input-group-sm"
+  - ""
+  - "input-group-lg"
+---
+
+<style>
```

Comment:
@mdo: you need to move this to docs CSS file.

Reply:
Yup yup, just putzing around for now :).